build_pbf_glyphs/src/main.rs

```diff
@@ -33,14 +33,29 @@ use std::sync::atomic::{AtomicUsize, Ordering};
 use std::thread;
 use std::time::Instant;
 
-use clap::{command, crate_authors, crate_description, crate_version, Arg};
+use clap::{command, Parser};
 use pbf_font_tools::freetype::{Face, Library};
 use pbf_font_tools::{get_named_font_stack, glyph_range_for_face, Glyphs};
 use protobuf::{CodedOutputStream, Message};
 use spmc::{channel, Receiver};
 
 static TOTAL_GLYPHS_RENDERED: AtomicUsize = AtomicUsize::new(0);
 
+#[derive(Parser, Debug)]
+#[command(version, author, about, long_about = None)]
+struct Args {
+    /// Sets the source directory to be scanned for fonts.
+    font_dir: String,
```

Comment:
You don't want to use `String` here - all paths are better used with `PathBuf` -- Clap supports them just fine

Reply:
Ahh TIL Clap supports those!